plugins/grafana-custom-plugins/grafana-chord-plugin/src/ChordPanel.test.tsx

```diff
@@ -41,14 +42,26 @@ describe('Chord Panel Test', () => {
     props.theme = createTheme();
 
     it('Should render chord', () => {
-        const arcSpy = jest.spyOn(d3, 'arc');
-        const ribbonSpy = jest.spyOn(d3, 'ribbonArrow');
-        const scaleOrdinalSpy = jest.spyOn(d3, 'scaleOrdinal');
-        const wrapper = mount(<UnthemedChordPanel {...props} />);
-
-        expect(wrapper.find(UnthemedChordPanel).exists()).toEqual(true);
-        expect(arcSpy).toHaveBeenCalled();
-        expect(ribbonSpy).toHaveBeenCalled();
-        expect(scaleOrdinalSpy).toHaveBeenCalled();
+        const { unmount } = render(<UnthemedChordPanel {...props} />);
```

Comment:
The naming of `unmount` is bit confusing to me. Could you help explain why we call `unmount()` in L64 will do the clean up?

Reply:
That is a feature when we use @testing-library/react.
https://testing-library.com/docs/react-testing-library/api/#render-result

The unmount function returned by the render function in the @testing-library/react library is a function that you can call to unmount (remove) the React component from the DOM and perform necessary cleanup.

In React components that use the useEffect hook, we can specify a cleanup function as the return value of the useEffect callback. This cleanup function is executed when the component is unmounted.

```
useEffect(() => {
  // Effect code

  return () => {
    // Cleanup code (executed when the component is unmounted)
  };
}, [dependencies]);
```